tests/suites/1_stateful/05_formats/05_02_csv/05_02_05_csv_with_0x01_delimiter.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../../shell_env.sh
+
+ echo "drop table if exists test_x01_csv" | $MYSQL_CLIENT_CONNECT
+
+ echo "CREATE TABLE test_x01_csv
+ (
+     a VARCHAR,
+     b Int,
+     c VARCHAR
+ );" | $MYSQL_CLIENT_CONNECT
+
+ curl -sH "insert_sql:insert into test_x01_csv format CSV" -H "format_field_delimiter:\x01" -H "format_skip_header:1" -H "format_record_delimiter:\r\n" -F "upload=@${CURDIR}/x01_field_delimiter.csv" \
```

Comment:
how about set format_record_delimiter to like \x02

Reply:
```
pub fn check_record_delimiter(option: &mut String) -> Result<()> {
    if option.is_empty() {
        *option = "\n".to_string()
    } else {
        let o = option.as_str();
        if o != "\n" && o != "\r\n" {
            return Err(ErrorCode::InvalidArgument(
                "record_delimiter can only be '\\n' or '\\r\\n'",
            ));
        };
    }
    Ok(())
}
```
I think we can remove the else check.

Reply:
This maybe the other PR, cc @youngsofun